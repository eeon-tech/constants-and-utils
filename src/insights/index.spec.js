const insights = require('./index')

describe('insights', () => {
  describe('crossingMarketInsightDirections', () => {
    it('should remain consistent', () => {
      expect(insights.crossingMarketInsightDirections).toMatchSnapshot()
    })
  })

  describe('sectorIndices', () => {
    it('should remain consistent', () => {
      expect(insights.sectorIndices).toMatchSnapshot()
    })
  })

  describe('marketIndices', () => {
    it('should remain consistent', () => {
      expect(insights.marketIndices).toMatchSnapshot()
    })
  })
})
