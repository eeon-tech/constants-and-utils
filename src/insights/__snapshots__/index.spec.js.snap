// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`insights crossingMarketInsightDirections should remain consistent 1`] = `undefined`;

exports[`insights marketIndices should remain consistent 1`] = `
Array [
  Object {
    "symbol": "DIA",
  },
  Object {
    "symbol": "IWM",
  },
  Object {
    "symbol": "QQQ",
  },
  Object {
    "symbol": "SPY",
  },
]
`;

exports[`insights sectorIndices should remain consistent 1`] = `
Array [
  Object {
    "symbol": "XLB",
  },
  Object {
    "symbol": "XLC",
  },
  Object {
    "symbol": "XLE",
  },
  Object {
    "symbol": "XLF",
  },
  Object {
    "symbol": "XLI",
  },
  Object {
    "symbol": "XLK",
  },
  Object {
    "symbol": "XLP",
  },
  Object {
    "symbol": "XLRE",
  },
  Object {
    "symbol": "XLU",
  },
  Object {
    "symbol": "XLV",
  },
  Object {
    "symbol": "XLY",
  },
]
`;
